tinder.js

```diff
@@ -339,6 +361,37 @@ function TinderClient() {
       makeTinderCallback(callback));
   };
 
+
+  /**
+   * Post new user picture
+   * @param {Buffer} buffer picture that you want to upload
+   * @param {Function} callback the callback to invoke when the request completes
+   */
+  this.uploadPicture = function(buffer, callback) {
+    tinderPost('image?client_photo_id=ProfilePhoto' + Math.floor(Date.now() / 1000),
```

Comment:
Can you compare the timestamp with your sniffed output? I think I recall it was epoch in milliseconds - you might need to use new Date().getTime() instead.